docs/src/examples/components/Portal/Types/PortalExampleControlled.shorthand.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+import { Button, Divider, Header, Label, Portal } from '@stardust-ui/react'
+
+class PortalExampleControlled extends React.Component<any, any> {
+  public state = {
+    log: [],
+    logCount: 0,
+    open: false,
+  }
```

Comment:
In effort to minimize lines, this object can be single line:

```jsx
state = { log: [], logCount: 0, open: false }
```

Reply:
done